src/app/content/utils/seoUtils.ts

```diff
@@ -25,6 +27,54 @@ const getParentPrefix = (node: LinkedArchiveTreeNode | undefined): string => {
 
 };
 
+const hideMath = (node: any) => {
```

Comment:
@TomWoodward what is the correct type for a DOM node/HTML element? TS complained about everything I tried.

Reply:
import HTMLElement from that openstax place